standards/git-workflow.md

```diff
@@ -0,0 +1,199 @@
+# Git workflow
+
+All repos MUST identify their "Git workflow" (aka "Merge strategy", "Steps for promoting a feature") in their README. A Git workflow identifies:
+
+ * the persistent branches (e.g. "development", "qa", "master"/"main"\*), including what environment they are deployed to if any
+ * what branch to cut feature branches from
+ * whether and how one bumps the component version (if it deviates from [the standard](./versioning.md#typical-workflow))
+ * any deviations from [our typical CI/CD configuration](https://github.com/NYPL/engineering-general/blob/updating-header/standards/travis-ci.md)
+
+Repos SHOULD link to the relevant Git workflow in this document and need not say anything more about the procedure if this document adequately describes it.
+
+For example:
+
+```
+## Contributing
+
+This repo uses the [Development-QA-Main Git Workflow](https://github.com/NYPL/engineering-general/tree/master/standards/git-workflow#development-qa-main)
+```
+
+## A note on master/main
+
+Repos should use "main" over "master". The workflows described in this document favor the term "main" over "master" anticipiting that all NYPL repos will eventually make that shift.
+
+## Common worklows
+
+The following are common Git workflows, named here so they can be easily referenced by the components that use them.
+
+### Development-QA-Main
+<a id="development-qa-master"></a>
```

Comment:
`main` in the id instead of `master`.

Reply:
I'm adding that for (temporary) backwards compatibility. There are repos out there that link to this document using the #development-qa-master hash and I don't want to break that, so adding an explicit 'development-qa-master' anchor to accompany the auto-generated 'development-qa-main' anchor that the heading above it generates for free.